app/components/show-header.js

```diff
@@ -0,0 +1,45 @@
+import Component from '@ember/component';
+import {get, set} from '@ember/object';
+
+function isAboveViewport(element) {
+    var rect = element.getBoundingClientRect();
+    return rect.top > 0;
+  }
+
+  let triggerStickyClass = function() {
+    let sentinel = document.querySelector(".new-donate");
+
+    if (!isAboveViewport(sentinel)) {
+        let denim = [0, 81, 142];
+        document.getElementById('sticky-donate').style.visibility = 'visible';
+        document.getElementById('listen-playhistory').style.backgroundColor = 'rgb(' + [...denim].join(',') + ')';
+    } else {
+        document.getElementById('sticky-donate').style.visibility = 'hidden';
+        document.getElementById('listen-playhistory').style.backgroundColor = 'transparent';
+    }
+  };
+  
+  export default Component.extend({
+      classNames: ['show-header'],
+      didRender() {
+          this._super(...arguments);
+          this.sendAction('updateParent', {
+              isHomepageForShow: get(this, 'isHomeForShow')
+          });
```

Comment:
I'm not seeing `isHomepageForShow` anywhere else in this PR. is it used in the app? You may not need to send this action